projects/approved-premises-and-delius/README.md

```diff
@@ -16,3 +16,27 @@ service when interacting with the probation case via Delius.
 ## Context Maps
 
 ![Context Maps](./tech-docs/source/img/approved-premises-and-delius-context-maps.png)
+
+## Interfaces
+
+### Message Formats
+
+The service responds to various HMPPS Domain Event message via the
+[Approved Premises and Delius Queue](https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-probation-integration-services-prod/resources/approved-premises-and-delius-queue.tf).
+The events are raised by the [HMPPS Approved Premises Service](https://github.com/ministryofjustice/hmpps-approved-premises-api)
+to communicate important events in the approved premises referrals process.
+
+Example [messages](./src/dev/resources/messages/) are in the development source tree.
+
+Incoming messages are filtered on `eventType` by the [SQS queue policy](https://github.com/ministryofjustice/cloud-platform-environments/blob/cc44e15d883b04d1caf5663eec6025674dc10eb5/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-probation-integration-services-prod/resources/approved-premises-and-delius-queue.tf#L5-L14)
+
+## Event Triggers
+
+Approved Premises domain events are raised in real time as approved premises
+referrals are processed. The progressive stages of a referral are raised as
+separate events
+
+| Business Event                                        | Message Class      | Message Event Type / Filter               |
+|-------------------------------------------------------|--------------------|-------------------------------------------|
+| A referral to an approved premises has been assessed  | HMPPS Domain Event | "approved-premises.application.assessed"  |
+| A referral to an approved premises has been submitted | HMPPS Domain Event | "approved-premises.application.submitted" |
```

Comment:
Do we need to mention the other 4 (e.g. booking.made)?  Or have those been left out intentionally because they haven't started sending them yet?

Reply:
Yes I'll add the subsequent events as they are implemented by the AP team